cmd/newrelic/main.go

```diff
@@ -8,13 +8,14 @@ import (
 	"github.com/newrelic/newrelic-cli/internal/config"
 	"github.com/newrelic/newrelic-cli/internal/credentials"
 	"github.com/newrelic/newrelic-cli/internal/entities"
+	"github.com/newrelic/newrelic-cli/internal/version"
 )
 
 var (
 	// AppName for this CMD
 	AppName = "newrelic-dev"
 	// Version of the CLI
-	Version = "dev"
+	Version = version.Version
```

Comment:
We don't need to set another global here, just pass it to `Execute` on line 39, or better yet is to remove completely from `main.go` and put directly into `command.go` inside of `Command`, and fix `versionCmd` to reference as well.